### PR TITLE
slider menu feature for mobile users

### PR DESCRIPTION
**Pull request by devalibello**
This pull request adds a slider menu to the portfolio website

### CODE REVIEW

**Comment by didierganthier** [comment]
Hi @devalibello ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) yet the commit history only shows a single contributor. Make sure that everybody contributes to the project. 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -9,42 +9,55 @@
...
+						<div class="view-b">
+							<ul class="desktop-view">
+								<li class="not1 not">Portfolio</li>
+								<li class="not2 not">About</li>
+								<li class="not3 not">Contact</li>
+							</ul>
+						</div>
+					</nav>
+				</header>
+				<section class="main_body">
```
Comment (OyePriscilla) [inline index.html:46]: - [optional] Please kindly format these lines of code properly in accordance to HTML/CSS best [practises](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 👍 **Kindly go through your codebase carefully and make the necessary adjustment**. I think there are a couple of extensions that does that automatically; for example,` prettier in vscode.  `
 
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @devalibello  and @danifromecuador 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Mobile menu implemented ✔️ 
Correct part of the page displayed when any of the mobile menu options clicked ✔️ 
Add JavaScript linters ✔️ 

## Required Changes ♻️

- [ ] Please kindly arrange your **Built With** stacks (-HTML -CSS -Javascript) to be under the tech stack section according to the [README file template](https://github.com/microverseinc/readme-template), this will give your repo a professional outlook. ℹ️ 
- [ ] @devalibello  Please kindly be more detailed with in your PR summary, it suppose to be a summary of all that you implemented in the PR. 👍 Thanks.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (devalibello) [reply index.html:46]: Okay. Thanks for your feedback and advice. However, I saw a lot of `prettier extensions`. Do you mean prettier code formatter

Comment (OyePriscilla) [reply index.html:46]: > Okay. Thanks for your feedback and advice. However, I saw a lot of `prettier extensions`. Do you mean prettier code formatter

Yes, prettier code formatter. You're right.

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @devalibello , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All changes required solved :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**